index.html

```diff
@@ -91,10 +91,13 @@ <h2>Statement of Intent</h2>
 
 <section>
 <h2>Code</h2>
-<ol>
-  <li>Treat each other with <a>respect</a>, professionalism,
-    fairness, and sensitivity to our many differences and strengths, including
-    in situations of high pressure and urgency.</li>
+<p>To help us achieve and maintain these high standards, each individual participant is expected to share
+	responsibility for our work environment by adhering to the following behavioral guidelines:</p>
+	
+	<ol>
+		<li><strong>Respect.</strong> We are a large community of people who are passionate about our work, sometimes holding strong opinions and beliefs. Misunderstandings and disagreements do happen and, when conflicts arise, we are expected to resolve them in a highly professional manner, with dignity and respect, and keeping our passions in check.</li>
```

Comment:
Suggestion: s/highly professional manner/respectful manner
Reason: "professional" is often understood as "for payment as part of a regular income", which is not a relevant meaning here; the alternative meaning of "done to a standard" is not actually related to respectfulness at all, yet this paragraph is about respect.

I feel uncomfortable about the "keeping our passions in check" ending here. First, passion is mentioned at the beginning in relation to "passionate about our work", and there's no suggestion that due to a misunderstanding people should become less passionate about their work. Second, I wonder if attempting to control the level of passion is itself a cultural norm that we should not be commenting about here. It might make sense to replace this with a pointer to "how to have a respectful discussion" text, which should say to keep discussions based on fact, be non-judgemental about others, etc.

Reply:
Regarding "passions" and "being passionate" - this language is copied [with minor edits] from the PWETF draft - the context behind it (at the time of drafting) was that it is given that we are passionate about what we do, and as a result, we are emotionally attached to the ideas we produce and contribute - keeping these emotions (i.e. our passions) in check when discussing alternative ideas contributed by other participants and not letting them cloud our judgement is a good thing.

Reply:
Thanks for the background @vlevantovsky , that's really interesting. If I understand you correctly you make a distinction between "passion for work" and personal emotion, i.e. anger/sadness/attraction/repulsion etc directed towards one or more individuals as opposed to an evaluation of one or more technical options that might be expressed using emotional terms. 

Or indeed an attempt to get to a particular decision by means of emotional manipulation of others, which I think we would also want to avoid, largely. 

I would agree with that distinction, however the text currently does not express it.

(aside: I imagine we all have real world experiences in which a group member's proposal is not well accepted and the group needs to find a way to say no without loss of face in order to facilitate smooth running of the group and sometimes even to avoid that person filing an objection to another proposal in order to get their own reconsidered. Is this emotional manipulation? Probably the group did not act respectfully in arriving at this situation. Still, humans are complicated...)

Reply:
@nigelmegitt it is an interesting way of looking at human emotions at a larger scale (bringing individual / group dynamics into the picture), but this is not how I saw it initially - it was all about individual emotions. For example, when you think about someone else's proposal as a "good idea" - you are not emotionally vested in it and is likely to evaluate it on its face value. However, when someone considers their own proposal a "good idea" and you don't see it as such  - their reaction may be a convolution of objective and subjective beliefs that 
a) the proposal is worthy of additional consideration (a constructive approach based on facts), and
b) it _is_ a good idea, how come you don't get it? (an emotionally charged response triggered by "it's _my proposal_, I really want it to succeed")
This is the kind of emotional responses we wanted participants to recognize (and keep in check), the discussion of group dynamics on emotional level, or a manipulation attempt triggered by "you objected to my idea, I will object to yours" never been considered, although I've certain seen it happened.

Reply:
Well @vlevantovsky it seems we may need to add a new section!

If it helps clarify this further, I had another thought about this recently, which is that in a section about _respect_, asking people not to be _passionate_ is strange because passion is very far from the opposite of respect. 

I think the wording here is trying to say "We deal with each other with courtesy, respect and dignity, even when emotions are heightened, which can happen because of misunderstandings, conflicts or disagreements particularly amongst people who are passionate about their work."

In other words, we don't ask people to be less passionate, but if they are going to be passionate, that doesn't excuse them from dealing with others properly. And even if they are _not_ passionate, they _still_ have to deal with others properly!

Reply:
@nigelmegitt That is accurate. I am wondering if a rewording this paragraph would help. 

What about:

> Respect.  We are a large community of people who are passionate about our work, sometimes holding strong opinions beliefs. We strive to deal with each other with courtesy, respect, and dignity at all times. Misunderstandings and disagreements do happen. When conflicts arise, we are expected to resolve them maintaining that courtesy, respect, and dignity, even when emotions are heightened.

Reply:
nit: s/strong opinions beliefs/strong opinions **and** beliefs

Reply:
with s/strive to/are committed to PWECG agreed to change at 4/4/2019 meeting